enterprise_access/apps/core/constants.py

```diff
@@ -26,6 +26,7 @@
 CONTENT_ASSIGNMENT_ADMIN_READ_PERMISSION = 'content_assignment.has_admin_read_access'
 CONTENT_ASSIGNMENT_ADMIN_WRITE_PERMISSION = 'content_assignment.has_admin_write_access'
 CONTENT_ASSIGNMENT_LEARNER_READ_PERMISSION = 'content_assignment.has_learner_read_access'
+CONTENT_ASSIGNMENTS_ACKNOWLEDGE_PERMISSION = 'content_assignment_configuration.has_acknowledge_assignments_access'
```

Comment:
Ah, I had opted for the variable name to be about assignments, not the assignment configuration given the POST API acts on assignments, not assignment configuration objects. If we continue with that, I could update the permission as follows?:

```
CONTENT_ASSIGNMENT_ACKNOWLEDGE_PERMISSION = 'content_assignment.has_acknowledge_access'
```

Otherwise, I can pull in your suggestion. Curious what you think about the options :)

Reply:
hm, I thought the POST API location under the assignment configuration viewset and URL route?  I know it doesn't write to the assignment configuration, but in my mind the logical placement of the view feels like it takes precedence over the action it actually takes.

Reply:
>  I thought the POST API location was under the assignment configuration viewset and URL route? 

@pwnage101 It is under the assignment configuration viewset/URL route, but so are the other endpoints related to these `CONTENT_ASSIGNMENT_*_PERMISSION` constants. For example, the `LearnerContentAssignmentViewSet` relies on `CONTENT_ASSIGNMENT_LEARNER_READ_PERMISSION`, but the API route is a URL route for the assignment configuration, in a somewhat similar fashion as the `acknowledge-assignments` action endpoint is, e.g.:

```python
router.register(
    r'assignment-configurations/(?P<assignment_configuration_uuid>[\S]+)/assignments',
    views.LearnerContentAssignmentViewSet,
    'assignments',
)
```

Reply:
In my mind, `/api/v1/assignment-configurations/<uuid>/acknowledge-assignments/` is arguably an action of an assignment configuration, whereas `/api/v1/assignment-configurations/<uuid>/assignments` is a nested route for a different assignment-specific viewset with assignment-specific permissions.  I lean towards calling it an assignment configuration permission rather than an assignment permission because I personally value the route hierarchy more than what the endpoint actually does in this case, but above all I don't want this to be a blocker either!

Reply:
Fair enough! I don't feel strongly one way or another; I think your rationale is sound 😄 I've updated to make this an assignment configuration permission!